.github/workflows/build_library.yml

```diff
@@ -31,6 +31,12 @@ jobs:
         with:
           platforms: arm64
 
+      - if: matrix.build-arch == 'manylinux2014_aarch64'
+        run: |
+          scripts/docker-aarch64-crossbuild.sh
```

Comment:
Would it make sense to build the library always in the same container? That way, we could also skip setting up the rust toolchain in the manylinux one.

Reply:
The one reason I added this new step and then reused `manylinux2014_aarch64` for the next step only because python and all the python dependencies are installed inside that container and it just works.  
But I have nothing against  installing correct version of the python and build everything in this one container 👍

Reply:
That's not what I meant, sorry for the confusion. The two containers are perfectly fine, in fact that will make it easier to align the library build with the binary build once we start cross compiling. Note that we already use a centos 7 based container for the production build of the Relay binary.

What I meant is that you can use a docker build for the x86_64 library as well, not just aarch64. That way, there's just a single way to build the library, and that's through docker. That'll make the script / Dockerfile more parametric, but it also allows us to build linux libraries on macOS hosts down the road.